scripts/container-pre-test.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+dnf install -y libnl3-devel python3-libvirt python3-dateutil python3-setuptools python3-pip dnf-plugins-core \
+    python3-requests python3-cryptography python3-subscription-manager-rhsm subscription-manager
+dnf builddep -y virt-who.spec
```

Comment:
i'd rework these in three steps:
- install `dnf-plugins-core`, so `dnf builddep` works
- `builddep` from the spec file, to install all the BuildRequires from the spec file automatically
- manual install of only the test-only packages

also, when installing stuff, please disable the weak dependencies (recommends), so we don't get unnecessary packages (makes the installation faster, and avoid side effects)

Reply:
Problem is that `dnf builddep` does not work as expected for some reason. Many packages are just not installed and it is necessary to install them manually. I would like to depend only on `dnf builddep`, but it is not possible ATM.

Reply:
which ones? anything I can check to help with this?

Reply:
I mean I don't want to fix issue with `dnf builddep` as part of this PR.

Reply:
well, if you don't even explain what the problem is, hardly people can help you...